.github/workflows/ethicalcheck.yml

```
@@ -0,0 +1,69 @@
+    - cron: '22 16 * * 6'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  Trigger_EthicalCheck:
+    permissions:
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)